src/api/submissions/submissions.service.js

```diff
@@ -155,6 +155,7 @@ const getLcContactConfigFromArray = async (
               local_council_notify_emails:
                 urlLcConfig.local_council_notify_emails,
               local_council_email: urlLcConfig.local_council_email,
+              local_guidance_link: urlLcConfig.local_guidance_link,
```

Comment:
The parameter name is different, the frontend uses local_council_guidance_link, while here it is local_guidance_link